app/src/pages/inside/manualLaunchesPage/launchSidePanel/launchSidePanel.scss

```
@@ -131,8 +132,11 @@
   }
 
   .meta-value {
+    min-width: 0;
     color: var(--rp-ui-base-almost-black);
     font-size: 13px;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)